### ISSUE TITLE
add an optional fine tuned model id when performing a query once it passes to the LLM

### ISSUE THREAD

**Issue by converseKarl** [llm-tools/embedJs#29]
Might this be a useful feature to keep things more private as the query is passed to GPT's and other LLM's where the model is more tuned one the query moves from semantic search/rag then passed to the LLM. An optional LLM model ID would be a useful feature.

**Comment by adhityan** [comment]
I don't understand what is being discussed here. Could you rephrase it please?
